### ISSUE TITLE
Temporal models require $_temporal to be defined in model definition

### ISSUE THREAD

**Issue by iturgeon** [fuel/orm#222]
Temporal models require that you define $_temporal to an empty array despite having default values for all it's options.

Given the following: 

``` php
<?
class Model_Vehicle extends Orm\Model_Temporal
{
    protected static $_properties = [
        'id',
        'name',
        'temporal_start',
        'temporal_end',
    ];
}

$car = Model_Vehicle::find(1);
// Parse Error - Argument 1 passed to Orm\Query::_parse_where_array() must be of the type array, null given, called in /mnt/obo/fuel/packages/orm/classes/query.php on line 529 and define
```

The problem comes down to no defaults being chosen for $_temporal when it isn't set. 

The code that finds the defaults was borrowed from Orm/Model where it is always defined by child classes.

The most concise and graceful way I could come up with is to simply define  $_temporal to an empty array on Temporal_Model.

pull request inbound...

**Comment by WanWizard** [comment]
@stevewest, when you have time?

**Comment by emlynwest** [comment]
This was closed with #223?

**Comment by emlynwest** [comment]
It was
